### DISCUSSION TITLE
Installing app packages with npm

### DISCUSSION THREAD

**Discussion by eggsandbeer** [vercel/turborepo#587] [Help]
Hi! Thanks for publishing this super cool tool. I'm just starting to learn it and I think it's great.

I have an incredibly obvious and probably dumb question though.

My monorepo/turborepo is using the typical `apps` & `packages` workplace/folder structure found in your examples. When I navigate in the command line to the root of an app under the `apps` folder and run something like `npm i -S json-stringify-safe`, i get an error saying that any deps that are in my packages folder referenced with the `*` flag (say it's `"dx-components": "*",`) can't be found on the npm registry.

`npm ERR! 404  'dx-components@*' is not in this registry.`

`yarn add json-stringify-safe` works fine, but is there some npm thing I'm not doing right? 

Even just running `npm i` under `turborepo/examples/design-system/apps/docs` errors out. 

`npm ERR! 404  '@acme/core@*' is not in this registry.`

Why is this? Do I need to tell npm to use local as a registry when i'm adding things inside an app? How? Tell me how to use npm please! What is the best way to add app dependencies? Just use yarn? Any help is 100% appreciated, thank you very much!

**Comment by samchouse** [comment]
I'm pretty sure in order to use a local project in NPM you need to use [local paths](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#local-paths)

I think this is related: #572
